graph_creator/static/graph_creator/graphs.js

```diff
@@ -0,0 +1,101 @@
+(function() {
+var d3 = Plotly.d3;
+
+var WIDTH_IN_PERCENT_OF_PARENT = 80,
+    HEIGHT_IN_PERCENT_OF_PARENT = 35;
```

Comment:
These numbers are for Chart's height and width.
This is the basic data from the implementation example.